### ISSUE TITLE
Permanent Notification

### ISSUE THREAD

**Issue by diegoleondev** [Raja0sama/rn-foreground-service#88]
How can I make the notification undeletable?

**Comment by Pauligrinder** [comment]
I don't think you can. Android 14 has changed the behaviour such that the previously permanent notifications can also be dismissed. On older versions I think the parameter `ongoing=true` should do the trick.

**Comment by Jamal-Ud-Din-hub** [comment]
Android 13+ `ongoing=true` **permanent** not working
